plugins/bufbuild/knit-ts/v0.0.7/Dockerfile

```diff
@@ -3,8 +3,6 @@ FROM node:20.11.1-alpine3.19 AS build
 WORKDIR /app
 COPY --link package*.json .
 RUN npm ci \
- && find node_modules/typescript ! -name 'typescript.js' ! -name 'package.json' -type f -exec rm -f {} + \
- && find node_modules/typescript -depth -type d -empty -delete \
```

Comment:
Still packaged as a separate layer but we are no longer trimming because this plugin requires transpile support (all the others don't).